jupyter-js-widgets/src/widget_layout.ts

```diff
@@ -112,7 +112,7 @@ class LayoutView extends DOMWidgetView {
         this._traitNames.forEach((trait) => {
             this.displayed.then((parent) => {
                 if (parent) {
-                    parent.el.style[this.css_name(trait)] = '';
+                    parent.el.style[this.css_name(trait)] = css_properties[trait];
```

Comment:
I was wondering if we should have a private `set_css(el, name, value)` which in fact does `el.style.removeProperty(name)` is `value` is an empty string.